src/common/packager.ts

```diff
@@ -43,7 +45,7 @@ export class Packager {
                             let args = ["--port", Packager.PORT];
                             let childEnvForDebugging = Object.assign({}, process.env, { REACT_DEBUGGER: "echo A debugger is not needed: " });
 
-                            Log.logMessage("Starting Packager", outputChannel);
+                            Log.logMessage("Starting Packager");
```

Comment:
We could pass a custom logger to the packager constructor and use the `logWithCustomLogger` instead.